growth/too/tasks/tiles.py

```diff
@@ -264,7 +264,9 @@ def get_planned_observations(
         field_ids, ras, decs, probs, nexposures = [], [], [], [], []
         segmentlist = segments.segmentlist()
         totprob = 0.0
-        for field_id in tile_structs[telescope].keys():
+
+        field_maps = {}
+        for ii, field_id in enumerate(tile_structs[telescope].keys()):
```

Comment:
How about:

```python
for tile_struct in tile_structs[telescope].values():
    ....
```
```

Reply:
Well we till need the field_id's themselves to keep track of below.

Reply:
You need the field_ids that are automatically assigned by PostgreSQL.

Reply:
Well I still need to know how to translate from the old one to the new one (to pick the right field_id from the scheduler).

Reply:
Are you concerned that you need `field_id` to create the associated `PlannedObservation` record? That's not the case. You can leave out the field_id in the constructors for both ORM classes, and SQLAlchemy/PostgreSQL will fill it in for you automatically. See https://stackoverflow.com/questions/27497666/sqlalchemy-autoincrement-value-as-foreign-key-mysql.

Reply:
Yes this was my assumption. Anyways, this goes beyond just the new galaxies Fields, but we use it for the "usual" tiling method, so I think this is still the right way to do it.